tests/click_tests/reset.py

```diff
@@ -116,12 +116,16 @@ def set_selection(debug: "DebugLink", diff: int) -> None:
 def read_words(debug: "DebugLink", do_htc: bool = True) -> list[str]:
     words: list[str] = []
 
+    # introductory screen
     if debug.layout_type is LayoutType.Caesar:
         debug.press_right()
     elif debug.layout_type is LayoutType.Delizia:
         debug.swipe_up()
     elif debug.layout_type is LayoutType.Eckhart:
-        debug.click(debug.screen_buttons.ok())
+        # In SLIP39, the screen is present only before the 1st share
```

Comment:
nit: maybe pass a share index as an additional argument to this function?
(to explicitly skip the instructions' screen before the first share)